markdown/chapters/meandeck/deckconstruction.md

```diff
@@ -0,0 +1,603 @@
+
+Proactive interaction is most commonly found in Doomsday lists as either
+discard effects such as Duress and {{Thoughtseize}} or tools like
+{{Teferi, Time Raveler}}. They are generally useful in every matchup
+but can be considered slow as they normally cost mana and are at
+Sorcery speed meaning they will not help prevent turn one plays when
+on the draw.
+
+Discard effects are good at providing information about a matchup early on and either 
+preventing your opponent landing relevant threats or taking away 
+their own interaction from them. They tend to excel in {U} based
+fair matchups or against opposing combo decks. They
+are often weaker against fair non-{U} decks or *stompy* type shells.
+The main limitation of discard effects is that, if they are used on a turn
+prior to the combo turn, the opponent may have a chance to draw into what
+they need. 
+
+Almost all styles of Doomsday list will run some number
+of discard effects, normally splitting between Duress and Thoughtseize
+and ranging between two and five copies. The ratio of the split normally
+favours Duress due to the impact of the life loss from Thoughtseize
+and the bias of non-creature spells being more impactful in matchups
+however this is meta-dependant and sometimes it is better to favour
+Thoughtseize. You should always include at least one of each in
+order to be able to incorporate the appropriate one in pile
+building. Some have tried alternatives like {{Cabal Therapy}}
+and {{Inquisition of Kozilek}} however they have been found
+to be not as effective as Duress and Thoughtseize.
+
+Teferi is a very strong tool in the Doomsday arsenal. It prevents almost
+all interaction during your combo turn and allows you to easily deploy
+your threats whilst disrupting theirs with minimal fear of reprisal. It
+also has the handy ability to act as slow card advantage and removal for
+troublesome permanents with the -3 ability. The main weaknesses of
+Teferi include the fact that it is a three mana spell which means it can
+be slow to be deployed. It is also removable once down through opposing
+cards like {{Abrupt Decay}} and requires you to be splashing into {W}.
+Because of this Teferi is not favoured outside of slower, more controlling
+lists in general.
+
+If running Teferi, it will normally be as a one to two of in the deck and
+will be run alongside discard effects and other reactive disruption.
+
+There is one more main class of proactive disruption in the form of running
+a creature package. The most commonly found card for this is {{Baleful Strix}}
+however other options like {{Uro, Titan of Nature's Wrath}} and {{Sedgemoor
+Witch}} have been tried. The idea behind these is they act as delays to
+your opponent's pressure whilst also providing additional card draws.
+In the case of Uro and Sedgemoor, they can also act as full plan B's
+providing an alternative way to win that isn't Doomsday.
+
+If running Strixes, these normally are swapped out for some mix of
```

Comment:
This sentence reads a bit confusing, imo

Reply:
Amended